content/docs/command-reference/destroy.md

```diff
@@ -57,7 +57,7 @@ $ ls -a
 .git code.py foo
 ```
 
-## Example: External cache directory
+## Example: External cache directory (is preserved)
```

Comment:
hmm, not sure I understand this?

Reply:
My intention was to clarify what the example is about. It shows that `destroy` doesn't affect external cache dirs. TBH I'm not sure this is the best place to show that, maybe just a brief note in the description?

Reply:
UPDATE: I see the note already exists. Changed to "Preserve an external cache directory" for now. I guess removing the example would be out of scope for this PR.

Reply:
UPDATE 2: Shoot, I went ahead and rephrased most of the example. It wasn't very clear before. Should I split into another PR @shcheklein ? See 62b4fed4369bf11fa4d6a4ccb2de1746d06a7fdf